lib/lotus/validations/attribute.rb

```diff
@@ -208,6 +209,17 @@ def nested
         end
       end
 
+      # Validates user defined validations
+      #
+      # @api private
+      def user_defined
+        @validations.each do |name, validation|
+          break unless validation.respond_to? :call
```

Comment:
The problem with that is that `@validations` may not only contain user defined validations. The build in vaildations are also in this hash and they are not callable objects (If I remember right).